docs/tutorials/MenuGuide.md

```diff
@@ -0,0 +1,237 @@
+            [SendClientMessage](../scripting/functions/SendClientMessage)(playerid, 0xFFFFFFFF, "Welcome to the top of Starfish Tower");
+        }
+        case 2: //Wheel Arch Angels
+        {
+            [SetPlayerPos](../scripting/functions/SetPlayerPos)(playerid, \-2705.5503, 206.1621, 4.1797);
+            [SetPlayerInterior](../scripting/functions/SetPlayerInterior)(playerid, 0);
+            [SendClientMessage](../scripting/functions/SendClientMessage)(playerid, 0xFFFFFFFF, "Welcome to the Wheel Arch Angels tuning-shop");
+        }
+        case 3: //Jizzys
+        {
+            [SetPlayerPos](../scripting/functions/SetPlayerPos)(playerid, \-2617.5156, 1390.6353, 7.1105);
+            [SetPlayerInterior](../scripting/functions/SetPlayerInterior)(playerid, 0);
+            [SendClientMessage](../scripting/functions/SendClientMessage)(playerid, 0xFFFFFFFF, "Welcome to Jizzy's Nightclub!");
+        }
+        case 4: //4Dragons
+        {
+            [SetPlayerPos](../scripting/functions/SetPlayerPos)(playerid, 2028.5538, 1008.3543, 10.8203);
+            [SetPlayerInterior](../scripting/functions/SetPlayerInterior)(playerid, 0);
+            [SendClientMessage](../scripting/functions/SendClientMessage)(playerid, 0xFFFFFFFF, "Welcome to the Four Dragons Casino");
+        }
+        case 5: //Com-a-Lot
+        {
+            [SetPlayerPos](../scripting/functions/SetPlayerPos)(playerid, 2169.1838, 1122.5426, 12.6107);
+            [SetPlayerInterior](../scripting/functions/SetPlayerInterior)(playerid, 0);
+            [SendClientMessage](../scripting/functions/SendClientMessage)(playerid, 0xFFFFFFFF, "Welcome to the Come-a-Lot casino!");
+        }
+    }
+}
+
+
+
+### 5 Last steps
+----------
+
+Now we need a command to show the menu. This is the easiest step. Just a comparison with strcmp and a ShowMenuForPlayer. As you should know, this comes in the callback OnPlayerCommandText
+
+if([strcmp](../scripting/functions/strcmp)(cmdtext, "/teleport", true) == 0)
+{
+    [ShowMenuForPlayer](../scripting/functions/ShowMenuForPlayer)(teleportmenu,playerid);
+    return 1;
+}
+
+Really easy, wasn't it?
+
+
+### 6 Last words
+----------
+
+Okay, after you read this AND understood it, try your own menu. As you could see, it isn't that difficult, but will impress the players on your server all the more. And you can script really cool effects with this. It's also very cool for general stores or supermarkets for the things you can buy. Then you can subtract some money as effect and the price is shown in another column in the menu. But now, work on your own.
+
+You can aslo add [TogglePlayerControllable](../scripting/functions/TogglePlayerControllable)(playerid,0); after ShowPlayerMenu(..);
+and  [TogglePlayerControllable](../scripting/functions/TogglePlayerControllable)(playerid,1) at end of OnPlayerSelectedMenuRow(..) so that player may not move while he is browsing menus.
```

Comment:
```suggestion
and  [TogglePlayerControllable](../scripting/functions/TogglePlayerControllable)(playerid,1) at the end of OnPlayerSelectedMenuRow(..) so that player may not move while they are browsing menus.
```